Assets/_nathan/scripts/GlobalMusic.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using _nathan.scripts;
+using UnityEngine;
+
+public enum EMusicTrackName
+{
+    MTN_Normal,
```

Comment:
What is `MTN_` ?

Reply:
standard ue4 naming convention for enums, [M]usic[T]rack[N]ame. I can remove it if you like

Reply:
removed from current pr